utils/spdx/src/main/resources/simple-license-mapping.yml

```diff
@@ -23,41 +23,22 @@
 ---
 
 AFLv2.1: AFL-2.1
-ALv2: Apache-2.0
```

Comment:
commit-msg: I believe the title should make clear that entries from the mapping are removed.